src/Components/Game.js

```diff
@@ -2,11 +2,12 @@ import { useState, useEffect } from "react";
 import axios from "axios";
 import GameOverDisplay from "./GameOverDisplay";
 import QuestionsDisplay from "./QuestionsDisplay";
+import Score from "./Score";
 
 const Game = (props) => {
   //Examples of Homophone Pairs https://www.englishclub.com/pronunciation/homophones-list.htm
   
-  const { initialWords } = props;
+  const { initialWords} = props;
```

Comment:
Looks like we lost a space on the right side of initialWords

Reply:
got it!